interview-questions.md

```diff
@@ -1,55 +1,37 @@
-# ASSESSMENT 1: Tech Interview Practice Questions
-
+# ASSESSMENT 1: Tech Interview Practice Questions --
```

Comment:
When answering these interview questions, please remember this is your practice for interviews. You appear to understand the terminology. Do the research and if you see that your answer is correct, find something interesting that you did not know before. Paraphrase and use your own words. Next week, let's focus on addressing the definition of the topic and something new that you found out.